packages/@react-stately/data/src/useTreeData.ts

```diff
@@ -51,6 +51,7 @@ export interface TreeData<T extends object> {
    */
   getItem(key: Key): TreeNode<T> | undefined,
 
+  getDescendantKeys(node?: TreeNode<T>): Key[],
```

Comment:
Does this need to be a public API? Maybe it could just be exported internally for the test?

Reply:
Agreed, we can make it public later if a use case comes up.